.vscode/settings.json

```diff
@@ -1,8 +1,8 @@
 {
-  "editor.formatOnPaste": true,
-  "editor.formatOnSave": true,
+  "editor.formatOnPaste": false,
+  "editor.formatOnSave": false,
   "editor.codeActionsOnSave": {
-    "source.fixAll": true
+    "source.fixAll": false
```

Comment:
The reason I have that, to have a common lint and formatting rule.

Reply:
Common lint is good, let the developer decide if he wants to use auto-formatting

Reply:
Yes, of course. We should agree on one otherwise there are too many changes in the next review ;)

Reply:
I'd say I can remove the the autoformatting from the settings.json file and you can enable those in your own settings ?